packages/common/src/failure.ts

```diff
@@ -263,16 +264,6 @@ export class ChildWorkflowFailure extends TemporalFailure {
   }
 }
 
-/**
- * Converts an error to a Failure proto message if defined or returns undefined
- */
-export function optionalErrorToOptionalFailure(
```

Comment:
These are _technically_ backwards-incompatible alterations right? I support removing them of course and I don't expect users ever used them.

Reply:
yup